### PR TITLE
Add amin/gs_check.sh to test new ghostscript versions for compliance

### PR DESCRIPTION
**Pull request by PaulWessel**
Given that ghostscript versions often are released with bugs (see #2903), we add this script so that prior to a new GMT release we can run this script with the very latest ghostscript version and see if basic features like fonts, patterns, and transparency are still converted correctly.

### CODE REVIEW

**Comment by PaulWessel** [comment]
This PR closes #2897.

File: admin/gs_check.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+if [ ! -d cmake ]; then
+	echo "Must be run from top-level gmt directory"
+	exit 1
+fi
+if [ ! -d build ]; then
+	mkdir -p build
+fi
+# Do the work in the build directory
+cd build
```
Comment (seisman) [inline admin/gs_check.sh]: Echo something here so that others know that the script outputs everything in the build directory.

Comment (PaulWessel) [reply admin/gs_check.sh]: OK will do

**Comment by PaulWessel** [comment]
None of the checks seem to finish the last few days...

**Pull request merged**